<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>1041</width>
      <height>621</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8pt &quot;Onyx&quot;;
font: 8pt &quot;Niagara Solid&quot;;
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>metier1</string>
     </attribute>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>1071</width>
        <height>611</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/emna_back.png</pixmap>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>160</y>
        <width>351</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>90</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Niagara Engraved&quot;;
</string>
      </property>
      <property name="text">
       <string>Fiche Patient</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>190</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>150</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>id_patient</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>290</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>240</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>290</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Rendez_vous</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>190</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>290</y>
        <width>171</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>experts</string>
     </attribute>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(0,0,0);
color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>480</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton


{
	
	background-color: rgb(72,78,86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
  
font: 81 10pt &quot;Rockwell Extra Bold&quot;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenom">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(0,0,0);
color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>250</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>-70</y>
        <width>351</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>250</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(0,0,0);
color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>480</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    
font: 81 10pt &quot;Rockwell Extra Bold&quot;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>310</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>specialite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>35</x>
        <y>-25</y>
        <width>1121</width>
        <height>631</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/emna_back.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Niagara Engraved&quot;;
</string>
      </property>
      <property name="text">
       <string>Ajout Des Experts </string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>130</y>
        <width>501</width>
        <height>221</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(255, 255, 255);
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>30</y>
        <width>271</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 28pt &quot;Niagara Engraved&quot;;
</string>
      </property>
      <property name="text">
       <string>Listes des experts</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>80</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;

font: 81 10pt &quot;Rockwell Extra Bold&quot;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
      </property>
      <property name="text">
       <string>Chercher</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_2">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>80</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(0,0,0);
color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>380</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton

{
	background-color: rgb(72, 78, 86);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
  
font: 81 10pt &quot;Rockwell Extra Bold&quot;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
      </property>
      <property name="text">
       <string>Trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>380</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton


{
	
	background-color: rgb(224,42,38);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
  
font: 81 10pt &quot;Rockwell Extra Bold&quot;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(106, 227, 231);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>40</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Downloads/bouton-daccueil (1).png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton

{
	background-color: rgb(217,193,187);
	border-style: outset;
    border-width: 2px;
    border-radius:20px;
    border-color: beige;
    
font: 81 10pt &quot;Rockwell Extra Bold&quot;;
    padding: 6px;
	color: rgb(255, 255, 255);

}

QPushButton:hover

{
	background-color: rgb(217,193,187);
	color: rgb(0, 0, 0);
	gridline-color: rgb(94, 118, 255);
}

QPushButton:pressed</string>
      </property>
      <property name="text">
       <string>HOME</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="specialite">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>310</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(0,0,0);
color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_supp">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>380</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(0,0,0);
color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="age">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>420</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 20px;
background-color: rgb(0,0,0);
color: rgb(255, 255, 255);
alternate-background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>370</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>sexe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>420</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
font: 81 10pt &quot;Rockwell Extra Bold&quot;;</string>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLabel" name="alerte_id">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>215</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="alerte_prenom">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>260</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="sp">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>380</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>-100</y>
        <width>1051</width>
        <height>721</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../Users/PC-GAMER/Downloads/emna_back.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="age_2">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>440</y>
        <width>261</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>380</y>
        <width>131</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>H</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>F</string>
       </property>
      </item>
     </widget>
     <zorder>label_6</zorder>
     <zorder>label_19</zorder>
     <zorder>id</zorder>
     <zorder>label_2</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>prenom</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>label</zorder>
     <zorder>nom</zorder>
     <zorder>pushButton</zorder>
     <zorder>label_5</zorder>
     <zorder>label_7</zorder>
     <zorder>tableView</zorder>
     <zorder>label_8</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>id_2</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>supp</zorder>
     <zorder>label_12</zorder>
     <zorder>pushButton_6</zorder>
     <zorder>specialite</zorder>
     <zorder>id_supp</zorder>
     <zorder>age</zorder>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
     <zorder>alerte_id</zorder>
     <zorder>alerte_prenom</zorder>
     <zorder>sp</zorder>
     <zorder>age_2</zorder>
     <zorder>comboBox</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1053</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
